<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoomBoxReceiver</class>
 <widget class="QWidget" name="BoomBoxReceiver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BoomBoxReceiver</string>
  </property>
  <widget class="QPlainTextEdit" name="inputPlainText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>421</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="visualizationTypelistView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
